integration/v3_grpc_test.go

```diff
@@ -193,3 +195,225 @@ func TestV3DeleteRange(t *testing.T) {
 		clus.Terminate(t)
 	}
 }
+
+// TestV3WatchFromCurrentRevision tests Watch APIs from current revision.
+func TestV3WatchFromCurrentRevision(t *testing.T) {
+	tests := []struct {
+		initialKeys   []string
+		initialValue  string
+		watchRequests []pb.WatchRequest
+		events        []storagepb.Event
```

Comment:
events -> wevents (want events). also put a blank line between the input and expected output.